### ISSUE TITLE
Feature: Display Profile data on Check-In-Page

### ISSUE THREAD

**Issue by mariav0** [progressivetech/net.ourpowerbase.qrcodecheckin#22]
Hi there,
I've thought that it would be a great feature to display profile data after on the Check-In-Page.
For example: You check in at an event and you want to tell this person directly their seat number instead of looking into another list.
I am not sure how difficult this would be to implement. Could you please estimate? I might get a funding for this.
Thanks in advance!

**Comment by jmcclelland** [comment]
Yes, sounds like a good idea. It's a bit hard to implement since the current display does not use a profile - it's hard coded. But, we could lookup the profiles in use by the given event page and pull all the data and display it as a block of text.

Unfortunately, I'm not available to do the work. I am happy to review any PR submitted by another developer.

**Comment by MegaphoneJon** [comment]
This isn't the solution, but if someone is desperate for this functionality, here's the code I used in one of my extensions:

```php
function myextensionname_civicrm_pageRun($page) {
  if ($page->getVar('_name') == "CRM_Qrcodecheckin_Page_QrcodecheckinLanding") {
    $programPreference = \Civi\Api4\Participant::get(FALSE)
      ->addSelect('Conference_Presenters_and_Invited_Guests_Questions.Check_In_Flags')
      ->addWhere('id', '=', $page->participant_id)
      ->execute()
      ->first()['Conference_Presenters_and_Invited_Guests_Questions.Check_In_Flags'] ?? NULL;
    if ($programPreference) {
      CRM_Core_Region::instance('page-footer')->add(array(
        'markup' => "<p>Check In Flags: $programPreference</p>",
      ));
    }
  }
}
```

**Comment by MegaphoneJon** [comment]
This functionality is present in the latest versions of QR Code Check-In - you can embed a FormBuilder display into your check-in form.

**Comment by jmcclelland** [comment]
Thanks @MegaphoneJon
